packages/cli/lib/base-generator.js

```diff
@@ -16,7 +14,6 @@ module.exports = class BaseGenerator extends Generator {
   // Note: arguments and options should be defined in the constructor.
   constructor(args, opts) {
     super(args, opts);
-    this._setupGenerator();
```

Comment:
Maybe it's better to remove redundant call of `this._setupGenerator()` in subclasses instead.